frontend/src/app/projects/project-detail/model-overview/move-model/move-model.component.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+import { Observable, map } from 'rxjs';
+import { ToastService } from 'src/app/helpers/toast/toast.service';
+import {
+  Model,
+  ModelService,
+} from 'src/app/projects/models/service/model.service';
+import {
+  Project,
+  ProjectService,
+} from 'src/app/projects/service/project.service';
+
+@Component({
+  selector: 'app-move-model',
+  templateUrl: './move-model.component.html',
+  styleUrls: ['./move-model.component.css'],
+})
+export class MoveModelComponent {
+  selectedProject?: Project;
+  search = '';
+  filteredProjects$: Observable<Project[] | undefined>;
+  constructor(
+    private modelService: ModelService,
+    private dialogRef: MatDialogRef<MoveModelComponent>,
+    private toastService: ToastService,
+    public projectService: ProjectService,
+    @Inject(MAT_DIALOG_DATA)
+    public data: { project_slug: string; model: Model }
+  ) {
+    this.projectService.loadProjectsForRole('manager');
+    this.filteredProjects$ = projectService.projects$.pipe(
+      map((projects) =>
+        projects?.filter((project) => project.slug !== data.project_slug)
+      )
+    );
+  }
+
+  onProjectSelect(project: Project) {
+    this.selectedProject = project;
+  }
+
+  async moveModelToProject(project: Project) {
+    this.modelService
+      .updateModel(this.data.project_slug, this.data.model.slug, {
```

Comment:
As there might be a few changes here when adding some UI changes, I'll only comment on this here. Depending on how you handled my previous comment, you should use the `this.modelService.moveModelToProjec` function here.